lib/standards/index.js

```diff
@@ -0,0 +1,31 @@
+import ariaAttrs from './aria-attrs';
+import { clone } from '../core/utils';
+
+const origAriaAttrs = clone(ariaAttrs);
+const standards = {
+	ariaAttrs
+};
+
+// @see https://stackoverflow.com/a/59008477/2124254
+function merge(current, updates) {
+	for (const key of Object.keys(updates)) {
+		if (!current.hasOwnProperty(key) || typeof updates[key] !== 'object') {
+			current[key] = updates[key];
+		} else {
+			merge(current[key], updates[key]);
+		}
+	}
+	return current;
+}
+
+export function configureStandards(config) {
+	if (config.ariaAttrs) {
+		standards.ariaAttrs = merge(ariaAttrs, config.ariaAttrs);
+	}
+}
+
+export function resetStandards() {
+	standards.ariaAttrs = origAriaAttrs;
```

Comment:
```suggestion
	standards.ariaAttrs = deepFreeze(origAriaAttrs);
```

Reply:
If we deep freeze then we can't merge in new properties. The standards `ariaAttrs` need to allow users to change it so we can pull it into other parts of the code and use the modified object.